.github/workflows/test.yml

```diff
@@ -41,6 +42,44 @@ jobs:
             stalled-download-timeout = 90
             timeout = 0
 
+      - name: Setup AWS Cache Keys
+        run: |
+          echo '${{ secrets.FLOX_STORE_PUBLIC_NIX_SECRET_KEY }}'  \
+               > /tmp/secret-key
+          {
+            printf 'extra-substituters = s3://flox-store';
+            printf '?secret-key=/tmp/secret-key&write-nar-listing=1';
```

Comment:
Do we need the secret-key to use the cache, or just for the nix copy at the end? Cause if we just need it in that step, I think it would be simpler to add most of this block to extra_nix_config, and just setup the secret key for the `Push Nix Store Contents` step

Reply:
I'm not sure. I just copied what I saw in `install-flox-action`.